docs/src/pages/components/radio-buttons/radio-buttons-de.md

```diff
@@ -43,6 +43,16 @@ Hier ist ein Beispiel zum Anpassen der Komponente. Mehr dazu erfahren Sie auf de
 
 {{"demo": "pages/components/radio-buttons/CustomizedRadios.js"}}
 
+## `useRadioGroup`
```

Comment:
Please leave the translated files untouched. We manage them in Crowdin.

Reply:
reset translated files
https://github.com/mui-org/material-ui/pull/21910/commits/d843b2270ad13ccf5e7b092e1fba8966029eec0f